### ISSUE TITLE
[SF][A11y][Popover] Content isn't read by screen reader when popover contains text only

### ISSUE THREAD

**Issue by Neeeko** [SAP/ui5-webcomponents#4738]
### **Bug Description**
When opening a ui5-popover which contains text only and no focusable element, the content of this popover is not announced by most screen readers. (NVDA will read it properly but not VoiceOver or JAWS).

![Screenshot 2022-02-11 at 15 27 44](https://user-images.githubusercontent.com/7316793/153609445-b3e205f3-b49a-4164-a759-89c7b1042e9b.png)

### **Expected Behavior**
When opening a popover which doesn't contain focusable element but textual content only, the content of the popover should be announced by the screen reader

### **Steps to Reproduce**
1. Using JAWS or VoiceOver (as NVDA announces it properly)
2. Open: https://codesandbox.io/s/ui5-webcomponents-forked-ys6b7?file=/index.html
3. Navigate to the button and use keyboard to open the popover
4. Content is not read

### **Context**
- UI5 Web Components version: 1.1.0
- OS/Platform: MacOS
- Browser: Google Chrome
- Affected component: Popover

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SuccessFactors

**Comment by ivoplashkov** [comment]
Hello @SAP/ui5-webcomponents-topic-rd, the issue is reproducible as described.
Could you please have a look?

Best regards,
Ivaylo

**Comment by rchrdchn** [comment]
Hey guys – just following up on this issue. When could we expect the fix for this issue?

**Comment by rchrdchn** [comment]
Following up here. Any updates?

**Comment by gmkv** [comment]
Hi,
We're currently working on this issue. We'll let you know when there's an update on the matter.

**Comment by GerganaKremenska** [comment]
Hello Neeeko,

 [Pull request 4517](https://github.com/SAP/ui5-webcomponents/pull/4517) intoroduces the way for static content to be read out by screen readers.

This functionality does not come out of the box as a WC can not know what is it in its content, we have provided accessibile-name-ref, so developers can link static content.

The feature is available from [v1.1.0](https://github.com/SAP/ui5-webcomponents/releases/tag/v1.1.0).

Here is an updated [code sample](https://codesandbox.io/s/ui5-webcomponents-forked-8k4ewu?file=/index.html) to use as a refferance.

Best Regards,
Gergana

**Comment by Neeeko** [comment]
Hi @GerganaKremenska,

The issue still seems to exist with JAWS where the content of the popover isn't read even with the `accessible-name-ref` attribute or using the code sample you provided - though it works with NVDA.

Can you please try to reproduce with JAWS?

Thanks!

**Comment by Neeeko** [comment]
Hi @gmkv and @ivoplashkov, can you please reopen this issue? As mentioned in my previous comment, the issue is still reproducible using JAWS.
Thanks!

**Comment by ilhan007** [comment]
Hello @GerganaKremenska @SAP/ui5-webcomponents-topic-rd  can you reiterate over the issue, the author has put some new comments after the provided solution.

**Comment by GerganaKremenska** [comment]
@Neeeko,

We are looking into the issue. As all accessibility standarts are met, this may take some time resolving.

--Gery

**Comment by GerganaKremenska** [comment]
Hello @Neeeko,

After a lot of investigation we found the "issue".
NVDA, Narattor and JAWS 2021 reads out everything as expected.

JAWS 2022 does not read out the content and that is an issue that we can not handle, because it comes from the vendor of the screen reader.

I am closing the issue as there is nothing on our side that we can do.

--Gery
